### ISSUE TITLE
Config: Use custom installed tslint

### ISSUE THREAD

**Issue by andy-hanson** [alm-tools/alm#257]
Version: 2.5.2

The error happens when using custom versions of tslint and typescript along with a custom lint rule.

``` sh
mkdir repro
cd repro
echo '{}' > tsconfig.json
echo '{ "devDependencies": { "tslint": "4.0.0-dev.0", "typescript": "2.0.3" } }' > package.json
npm install
mkdir tslint-bin
echo 'const Lint = require("tslint/lib/lint"); exports.Rule = class extends Lint.Rules.AbstractRule { apply() { return [] } }' > tslint-bin/myRule.js
echo '{ "rulesDirectory": [ "tslint-bin" ], "rules": { "ordered-imports": true, "my": true } }' > tslint.json
mkdir src
echo 'true ? 1 : undefined' > src/a.ts
node node_modules/tslint/bin/tslint src/a.ts # Works
alm -o
```

The error is:

```
TSCONFIG: All Good! /home/andy/repro/tsconfig.json
[TSC] Started Initial Error Analysis: /home/andy/git/work/sample/repro/tsconfig.json
[LINT] About to start linting files:  1
[LINT] Linter crashed TypeError: Cannot read property 'text' of undefined
    at Object.getTokenPosOfNode (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/typescript/lib/typescript.js:6279:71)
    at TokenObject.TokenOrIdentifierObject.getStart (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/typescript/lib/typescript.js:73033:23)
    at OrderedImportsWalker.visitNode (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/rules/orderedImportsRule.js:103:33)
    at /home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/language/walker/syntaxWalker.js:459:63
    at visitNode (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/typescript/lib/typescript.js:12816:20)
    at Object.forEachChild (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/typescript/lib/typescript.js:12978:21)
    at OrderedImportsWalker.SyntaxWalker.walkChildren (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/language/walker/syntaxWalker.js:459:12)
    at OrderedImportsWalker.SyntaxWalker.visitConditionalExpression (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/language/walker/syntaxWalker.js:52:14)
    at OrderedImportsWalker.SyntaxWalker.visitNode (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/language/walker/syntaxWalker.js:273:22)
    at OrderedImportsWalker.visitNode (/home/andy/.nvm/versions/node/v6.8.0/lib/node_modules/alm/node_modules/tslint/lib/rules/orderedImportsRule.js:109:36)
[TSC] Started Initial Error Analysis: /home/andy/repro/tsconfig.json
[TSC] Error Analysis Duration: 5s
[TSC] FileCount: 2  Errors: 0
[TSC] Error Analysis Duration: 2s
[TSC] FileCount: 2  Errors: 0
```

Notice that the stacktrace is using alm's version of tslint, not the locally installed one.

**Comment by basarat** [comment]
Rewording as a feature request of using a custom tslint. Should be easy enough as only https://github.com/alm-tools/alm/blob/master/src/server/workers/lint/lintWorker.ts will need changing + some way to provide a config.

**Comment by basarat** [comment]
Automatically picks up the linter using findup from the current project tsconfig.json, otherwise uses the local version :rose: 

Released with 2.11.1
